extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -369,16 +357,8 @@ export class SubscriptionService {
       const selectedPlatform = resolvedValues[1];
       const selectedEntitlement = this.platformStore_.getResolvedEntitlementFor(
           selectedPlatform.getServiceId());
-      /** @type {!RenderState} */
-      const renderState = {
-        entitlement: selectedEntitlement.json(),
-        loggedIn: selectedEntitlement.loggedIn,
-        subscribed: !!selectedEntitlement.subscriptionToken,
-        granted: grantState,
-        metered: !!selectedEntitlement.metering,
-      };
-
-      selectedPlatform.activate(renderState);
+
+      selectedPlatform.activate(selectedEntitlement.json());
```

Comment:
Let's just pass the entitlement itself. The platform can always call `json()` on it itself if it wants to :)